CMakeSettings.json

```diff
@@ -0,0 +1,66 @@
+﻿{
```

Comment:
Is this file supposed to be here? It looks like it may be a local configuration file that accidentally was added.  If not, it seems like it will constantly be outdated with paths such as `C:/Qt/5.15.2/msvc2019_64`, `C:/Strawberry/perl`.

Reply:
@dan-giddins that's a visual studio ide config file. Also, the commits you pushed now look ambiguous.